junit-jupiter-engine/src/main/java/org/junit/jupiter/engine/config/InstantiatingConfigurationParameterConverter.java

```diff
@@ -33,18 +35,23 @@ public InstantiatingConfigurationParameterConverter(Class<T> clazz, String name)
 	}
 
 	Optional<T> get(ConfigurationParameters configurationParameters, String key) {
+		return supply(configurationParameters, key).get();
+	}
+
+	Supplier<Optional<T>> supply(ConfigurationParameters configurationParameters, String key) {
```

Comment:
@marcphilipp although this is implicitly covered by existing tests of `get`, I assume we want to have dedicated tests for `supply` since it's exposed, right?

Reply:
Since they would only call `get` I'm not sure if it's worth it.

Reply:
I'm not sure either. I leave it aside.